server/utils/queryHelpers/scopeGet.js

```diff
@@ -68,6 +68,17 @@ const getActiveIds = ({ activePub, activeCollection, activeCommunity }) => {
 	};
 };
 
+const getActivePubQuery = ({ id, slug, communityId }) => {
+	if (!communityId && slug) {
+		throw new Error('Pub slug must be passed to scopeGet with a communityId');
+	}
+	return {
+		...(id && { id: id }),
+		...(slug && { slug: slug }),
+		...(communityId && { communityId: communityId }),
+	};
```

Comment:
Not strongly opinionated here but this could be cleaned up with a helper fn, e.g.

```js
return removeEmpty({
  id: id,
  slug: slug,
  communityId: communityId,
});
```

You don't win any points in terms of line count but is a bit easier to read. Feel free to ignore.

Reply:
Good idea! Calling it `stripUndefinedValues` because that's the behavior we actually need.

Reply:
pfft, wait, no it's not.